configuration/etl/etl_tables.d/hardware/hardware_types.json

```diff
@@ -38,6 +38,24 @@
                 "nullable": false,
                 "comment": "The amount of physical memory on the node in GiB"
             },
+            {
+                "name": "numa_node_count",
+                "type": "int(11)",
+                "nullable": false,
+                "comment": "The amount of NUMA nodes"
```

Comment:
This is a nitpick, but "The number of NUMA nodes" sounds more accurate. Same for disk and ethernet count.